__tests__/Model.test.js

```diff
@@ -9,6 +9,8 @@ const Schema = require('../Lab-TDD-With-Jest/starter-code/Schema');
 const Model = require('../Lab-TDD-With-Jest/starter-code/Model');
 
 //A model needs a schema to check against. The Schema defines the patter we expect. I.e. A name will be required and the input type will be a string. 
+
```

Comment:
Really like how far you have come with how clean your code is! No white space and everything named in an easy to follow way. Great job man!